src/ServiceControl/Infrastructure/Settings/RemoteInstanceSetting.cs

```diff
@@ -6,6 +6,11 @@
     public class RemoteInstanceSetting
     {
         public string ApiUri { get; set; }
+        /// <summary>
+        /// If for any reason we fail to connect to remove instance they will be disabled. Any <see cref="ScatterGatherApiBase"/> query will skip that instance. <see cref="CheckRemotes"/> custom check takes care of that. 
```

Comment:
```suggestion
        /// If we fail to connect to a remote instance, it will be temporarily disabled. Any <see cref="ScatterGatherApiBase"/> query will skip disabled instances. The <see cref="CheckRemotes"/> custom check will enable any disabled remote instance the first time it succeeds in connecting it. 
```